### ISSUE TITLE
Customize toolbar for jupyterlab notebooks 

### ISSUE THREAD

**Issue by SonakshiGrover** [jupyterlab/jupyterlab#8887]
### Problem

* I am using jupyterhub with jupyterlab.
* I have 2 folders: A and B. Folder A had notebooks which I don't want any user to execute, the user can just see the notebook but cannot write or execute it. Folder B has notebooks which user can read/write/execute. 

### Proposed Solution

* To achieve the above, my plan is to show a set of toolbar buttons when notebooks from folder A are opened and another set of toolbar buttons when notebooks from folder B are opened. So for example, when Folder A notebooks are opened, user can will not see run/save buttons and when Folder B notebooks are opened, user can see all the buttons.
* So basically, as an admin of jupyterhub, I want to customize the toolbar according to the folder location of the notebook currently being displayed.

Is there already a feature in jupyterhub/jupyterlab for this? If not, any suggestions to achieve the above would be very helpful for me. Thanks.
